index.html

```diff
@@ -213,29 +220,27 @@ <h2 class="component-header-box text-orange">Articles, Podcasts, & More</h2>
       <p class="text-blue font-family-serif font-size-large">Unfiltered & authentic takes on life</p>
     </div>
     <div class="col-sm-8">
-      {% assign below_the_fold = page.home_collection.docs | slice: 4 %}
-      {% for article in below_the_fold %}
+      {% assign most_recent = page.home_collection.docs | sort: "published_at" | reverse | first %} 
```

Comment:
I think the `reverse` filter is causing the newest article to be last and the 2nd newest first. Otherwise, this seems right to me

![screenshot 2020-06-22 at 15 50 39](https://user-images.githubusercontent.com/56852436/85329380-6aee7180-b4a0-11ea-9b99-c32acd8fdbeb.png)

Reply:
Thanks @dholloran-ample. I think the discrepancy is that you're sorting via `updated_at` while he's sorting via `published_at`.